src/fsharp/FSharp.Core.nuget/FSharp.Core.LatestNuget.nuspec

```diff
@@ -9,7 +9,7 @@
             FSharp.Core redistributables from Visual F# Tools version 10.1 For F# 4.1
 
                 Supported Platforms:
-                    .NET Framework 4.5+ (net45)
+                    .NET Framework 4.6+ (net46)
```

Comment:
Does FSharp.Core really need to move to 4.6?  I'm pretty sure we should keep this specific component targeting 4.5

Reply:
It's ok for the compiler components to target 4.6 though.

Reply:
Good catch; FSharp.Core doesn't reference any of the components that will eventually need to be 4.6.  I got overly aggressive with find/replace.